README.md

```diff
@@ -94,12 +94,32 @@ which you may need to install (we have some fallbacks if you don't):
 - [libsasl](https://cyrusimap.org/sasl/)
 
 To install these on debian you can type:
+
 ```
     sudo apt install libetpan-dev libssl-dev libsqlite3-dev libsasl2-dev libbz2-dev zlib1g-dev
 ```
 
+If you run into connection problems with IMAP (error #43) it might be the case that the `libetpan` version on your system is too old. In this case we recommend compiling `libetpan`:
```

Comment:
I'd refer to https://github.com/dinhviethoa/libetpan/pull/330 here so that we know why we're recommending this and when we can remove it or how to clarify.  In particular it's only relevant if your libetpan links against gnutls.